src/cbmc_viewer/tracet.py

```diff
@@ -329,7 +329,7 @@ def parse_xml_traces(xmlfile, root=None):
     if xml.find('result') is not None:
         for line in xml.iter('result'):
             name, status = line.get('property'), line.get('status')
-            if status == 'SUCCESS':
+            if status == 'SUCCESS' or status == 'UNKNOWN':
```

Comment:
Would that make any existing warning silent?

Reply:
No, there never was a warning about this, we now just have unsuccessful attempts to extract a trace when the status is not `SUCCESS`.